examples/mgnni/mgnni_trainner.py

```diff
@@ -0,0 +1,212 @@
+import os
+import time
+
+import numpy as np
+
+os.environ["KMP_DUPLICATE_LIB_OK"] = "TRUE"
+os.environ['TL_BACKEND'] = 'torch'
+# os.environ['TL_BACKEND'] = 'mindspore'
+# os.environ['TL_BACKEND'] = 'paddle'
+# os.environ['TL_BACKEND'] = 'tensorflow'  # set your backend here, default `tensorflow`
+os.environ["CUDA_VISIBLE_DEVICES"] = "0"
+import sys
+
+sys.path.insert(0, os.path.abspath('../../'))  # adds path2gammagl to execute in command line.
+import argparse
+import tensorlayerx as tlx  # 导入tensorlayerx包
+from gammagl.datasets import WebKB, WikipediaNetwork  # 专门做读取数据的类Planetoid
```

Comment:
Remember to replace Chinese comment with English.

Reply:
Thank you for your reminder. I apologize for the oversight in leaving Chinese comments in the code. I will make sure to replace them with English comments to ensure clarity and consistency.